bigtable/quickstart/README.md

```diff
@@ -0,0 +1,78 @@
+# Cloud Bigtable Quickstart in Go
+
+This is a simple application that demonstrates using the [Google Cloud APIs Go
+Client Library](https://github.com/GoogleCloudPlatform/google-cloud-go) to connect
+to a Cloud Bigtable instance and read a row from a table.
+
+## Before you begin
+
+1. [Select or create][projects] a Cloud Platform project.
+
+1. Enable [billing][billing] for your project.
+
+1. Enable the Cloud Bigtable API][enable_api].
+
+    Note: The quickstart performs an operation on an existing table.
```

Comment:
"an existing table (created below)."

There might be confusion of whether or not the user needs their own table from the beginning.

Reply:
done